web/css/source/_theme.less

```diff
@@ -195,6 +226,7 @@ p,li,dt,dd,td,th,details,button,form,a,div,details {
     .content-container{
       padding-left: 15%;
       display: block;
+      celear:both;
```

Comment:
Typo (clear not celear), so either this isn't needed and should be removed,  or it should do something that it currently isn't and should be fixed.